renderdoc/driver/d3d11/d3d11_context.cpp

```diff
@@ -1113,11 +1113,12 @@ void WrappedID3D11DeviceContext::AddUsage(const ActionDescription &a)
   //////////////////////////////
   // OM
 
-  for(int i = 0; i < D3D11_1_UAV_SLOT_COUNT; i++)
+  for(int i = pipe->OM.UAVStartSlot; i < D3D11_1_UAV_SLOT_COUNT; i++)
```

Comment:
No need to change anything. 
I wondered if a comment about the for loop and the indexing being used might help i.e. is the for loop in OM slots or pixel shader slots (as it is currently)

Another thought was perhaps the loop could be over the Output Merger slot range i.e. 0 -> `D3D11_1_UAV_SLOT_COUNT - pipe->OM.UAVStartSlot` then the pixel shader reference would be `i + pipe->OM.UAVStartSlot`. It is the same thing in the end for either for loop and other bits of code do not have comments and use the correct offset.

I would leave it as it is unless you have strong opinions and want to change it

Reply:
I'm OK with this change as-is. The annoying thing about this particular API wart is that both indices are useful in different cases, but here when we're looking primarily from the perspective of the bindings in the shader it seems natural to index with the 'register' and offset when looking up the stored list of UAVs.

I do agree that a comment is worthwhile since this has shown up with a bug now, at least to explain what's happening.